setup.py

```diff
@@ -43,7 +39,7 @@ def main(args):
             "Intended Audience :: Developers",
             "Topic :: Security :: Cryptography",
 
-            "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
+            'License :: OSI Approved :: MIT License',
```

Comment:
MIT requires to provide copy of the license to be shared with every instance of the application, which is inconvient.
This is the reason why I chose CC0 (public domain) back then.

Reply:
Neither `grammers-cryptg` or `pyo3` are CC0, though. Can we still use CC0 if we depend on non-CC0 libraries?

Reply:
> Can we still use CC0 if we depend on non-CC0 libraries?

Yes, as long as you share licenses for the works being used.
https://wheel.readthedocs.io/en/stable/user_guide.html#including-license-files-in-the-generated-wheel-file

Also please remove `build_ffi.py` and other stuff related to current approach based on `cffi`.

Reply:
> Also please remove `build_ffi.py` and other stuff related to current approach based on `cffi`.

Should I get rid of `appveyor.yml` and `.travis.yml` as well?

Reply:
> Should I get rid of `appveyor.yml` and `.travis.yml` as well?

Have you provided another way to automatically build wheels for all supported flavours of Windows / Linux / macOS?

Reply:
Oh, no, forgot Linux. I'll get rid of the GH Actions script, and we'll have to modify travis one, I have no idea how it works.

Reply:
Also please restore proper string punctuators on this line.
`"` means human-readable text, while `'` means identifier-like technical strings.